mon-pix/app/components/user-certifications-detail-header.js

```diff
@@ -0,0 +1,17 @@
+/* eslint ember/no-classic-components: 0 */
+/* eslint ember/require-tagless-components: 0 */
+
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import classic from 'ember-classic-decorator';
+import { inject as service } from '@ember/service';
+
+@classic
+export default class UserCertificationsDetailHeader extends Component {
+  @service intl;
+
+  @computed('certification.birthdate')
+  get birthdate() {
+    return this.intl.formatDate(this.certification.birthdate, { format: 'LL' });
+  }
+}
```

Comment:
Est-il possible de glimmeriser ce composant?
Cela permetrait d'enlever les commentaires au début du fichier et de supprimer le `classic`.
https://guides.emberjs.com/v3.18.0/upgrading/current-edition/glimmer-components/

(je n'ai pas testé la suggestion ci-dessous)
```suggestion
import Component from '@glimmer/component';
import { inject as service } from '@ember/service';

export default class UserCertificationsDetailHeader extends Component {
  @service intl;

  get birthdate() {
    return this.intl.formatDate(this.args.certification.birthdate, { format: 'LL' });
  }
}
```

Reply:
Si le composant est glimmerisé:
- côté js, les arguments du composant sont appelés via `this.args` (ex: `this.args.certification`) et les variables locales via `this`
- côté hbs, les arguments du composant sont appelés via `@` (ex: `@certification`) et les variables locales via `this`
- les arguments sont automatiquement `tracked` (`tracked` = `computed`)